src/lib/degradeableItems.ts

```diff
@@ -297,6 +298,17 @@ export async function degradeItem({
 	const degItem = degradeableItems.find(i => i.item === item);
 	if (!degItem) throw new Error('Invalid degradeable item');
 
+	// 5% chance to not consume a charge when Ghommal's lucky penny is equipped
```

Comment:
If its a 5% percent chance, we should use `percentChance(5)`

Reply:
Switched it to percentChance(5)